src/TTextEdit.cpp

```diff
@@ -1160,28 +1162,43 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
 
 
         QAction* action = new QAction("copy", this);
-        action->setStatusTip(tr("copy selected text to clipboard"));
+        action->setToolTip(tr("copy selected text to clipboard"));
```

Comment:
All this stuff is super-explanatory - tooltips just get in the way. I'd suggest removing them

Reply:
Ah, despite the Qt Documentation saying about `QAction`s:
> `toolTip : QString`
> This property holds the action's tooltip.
>
> This text is used for the tooltip. If no tooltip is specified, the action's text is used.
>
> By default, this property contains the action's text.

It looks like we can avoid a tooltip of `copy` popping up for this one (and similarly for _other_ ones) now that we have enabled tooltips on the parent `QMenu` (for all `QAction`s on the menu) by setting the unwanted tooltips to be empty manually...